pkg/users/group-details.js

```diff
@@ -0,0 +1,185 @@
+    Label, LabelGroup,
+    Page, PageSection,
+    Select, SelectOption, SelectVariant, Text, TextVariants, Breadcrumb, BreadcrumbItem,
+    Form, FormGroup,
+} from '@patternfly/react-core';
+import { UndoIcon } from '@patternfly/react-icons';
+
+import { GroupActions } from "./group-actions.jsx";
+
+const _ = cockpit.gettext;
+
+export function GroupDetails({ groups, accounts, group }) {
+    const current_group = groups.find(g => g.name == group);
+    return (
+        <Page groupProps={{ sticky: 'top' }}
+              isBreadcrumbGrouped
+              id="group"
+              breadcrumb={
+                  <Breadcrumb>
+                      <BreadcrumbItem to="#/">{_("Groups")}</BreadcrumbItem>
+                      <BreadcrumbItem isActive>{group}</BreadcrumbItem>
+                  </Breadcrumb>}>
+            <PageSection>
+                <Card className="group-details" id="group-details">
+                    <CardHeader>
+                        <CardTitle id="group-title"><Text component={TextVariants.h2}>{group}</Text></CardTitle>
+                        {superuser.allowed && <CardActions>
+                            <GroupActions group={current_group} accounts={accounts} />
+                        </CardActions>}
+                    </CardHeader>
+                    <CardBody>
+                        <Form isHorizontal onSubmit={apply_modal_dialog}>
+                            <FormGroup fieldId="group-name" hasNoPaddingTop label={_("Name")}>
+                                <output id="group-name">{group}</output>
+                            </FormGroup>
+                            <GroupAccountsSelect accounts={accounts} group={group} groups={groups} />
+                        </Form>
+                    </CardBody>
+                </Card>
+            </PageSection>
+        </Page>
+    );
+}
+
+export const GroupAccountsSelect = ({ accounts, group, groups }) => {
+    const current_group = groups.find(g => g.name == group);
+    const [isOpenAccount, setIsOpenAccount] = useState(false);
+    const [selected, setSelected] = useState([...current_group.userlist, ...current_group.userlistPrimary]);
+    const [history, setHistory] = useState([]);
+    const [loggedInUsersAffected, setLoggedInUsersAffected] = useState(false);
+
+    const undoAccountChanges = () => {
+        const undoItem = history[history.length - 1];
+        if (undoItem.type === 'added') {
+            removeAccount(undoItem.name, true).then(() => setHistory(history.slice(0, -1)));
+        } else if (undoItem.type === 'removed') {
+            addAccount(undoItem.name, true).then(() => setHistory(history.slice(0, -1)));
+        }
+    };
+
+    const removeAccount = (account, isUndo) => {
+        if (!isUndo)
+            setHistory([...history, { type: 'removed', name: account }]);
+
+        const current_account = accounts.find(a => a.name == account);
+        if (current_account.loggedIn)
+            setLoggedInUsersAffected(true);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230222-100737-2beed508-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)